gems/quilt_rails/app/controllers/quilt/performance_report_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Quilt
+  class PerformanceReportController < ActionController::Base
+    protect_from_forgery except: :index
+    include Quilt::Performance::Reportable
+
+    def index
+      process_report do |client|
```

Comment:
Why is there any customization here? Wouldn't a consumer expect that anything the default controller will do is all encompassed by the mixin? I can't say I understand how you decided what would be here versus in `process_report`.

Reply:
Yeah that makes sense, I made the defaults in `process_report` the same as in koa but I added here behaviour that made sense to me as defaults, probably should just throw the bits that are most obvious into the mixin itself and just do the call + response rendering here.